webxrlayers-1.bs

```diff
@@ -1965,7 +2031,7 @@ monoscopic devices.
 XRView changes {#xrviewchanges}
 --------------
 Each [=view=] MUST define a <dfn>recommended WebGL texture resolution</dfn> which represents a best estimate of the WebGL texture
-resolution large enough to contain the view.
+resolution large enough to contain the view and a <dfn>recommended motion vector texture resolution</dfn> which is based on the [=recommended WebGL texture resolution=].
```

Comment:
I can ask internally for more clarification. Since this is hot of the press, I'm sure we will create a blog post to go more into detail.
From talking to the team earlier, it sounds like motion vectors are usually generated in a separate pass and that the shaders are very simple. Even so, it would likely be expensive to write to a texture that's twice the size if they didn't constrain the size.